src/app/modules/shared/store/technology.actions.ts

```diff
@@ -0,0 +1,28 @@
+import { Action } from '@ngrx/store';
+import { Technology } from '../models/technology.model';
+
+export enum TechnologyActionTypes {
+  LOAD_TECHNOLOGY = '[Technology] LOAD_TECHNOLOGY',
+  LOAD_TECHNOLOGY_SUCCESS = '[Technology] LOAD_TECHNOLOGY_SUCCESS',
+  LOAD_TECHNOLOGY_FAIL = '[Technology] LOAD_TECHNOLOGY_FAIL',
```

Comment:
I think here we are not LOADING but FINDING so:
FIND_TECHNOLOGIES
FIND_TECHNOLOGIES_SUCESS
FIND_TECHNOLOGIES_FAIL